test/sourcemaps.js

```diff
@@ -31,14 +31,12 @@ var readExpectation = function(filename) {
   return fs.readFileSync('test/fixtures/sourcemaps-expectations/' + filename).toString().replace(/\n$/, '');
 };
 
-function writeTestOutput(done) {
+function writeTestOutput() {
   var builder = new Builder();
   return builder.loadConfig(configFile)
     .then(function() {
       builder.buildStatic('first.js', 'test/output/output.js', buildOpts);
-    })
-    .then(done)
-    .catch(err);
```

Comment:
I do not remember why I've deleted the `done` callback trigger. Maybe `done` was undefined.